plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,18 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works on Direct3D check git #1900
```

Comment:
You should explain the issue in the comment, not mention some git PR future coders might not be able to find. ;)